spec/factories/mail_aliases.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :mail_alias do
-    email { "#{Faker::Internet.user_name}@#{Rails.application.config.x.mail_domains.sample}" }
+    email { "#{Faker::Internet.user_name}@sandbox86621.eu.mailgun.org" }
```

Comment:
Are you specifically choosing the sandbox instead of 1 of the 3 `mail_domains`?
If so, thats fine, but it wasn't in the old code so maybe it's a mistake :)

Reply:
Yes, this way you do not have production addresses on staging or local and cannot accidentally activate a production SMTP from localhost